### PR TITLE
Add command to uninstall LaravelGitHooks definded git hooks

### PR DESCRIPTION
**Pull request by listefano**
## Description

This PR add the functionality described in issue #16 

I used the delimiter from the PR #19 to keep the user defined hooks intact.

## How has this been tested?

I used composer to symlink to local repository to a new Laravel project and tested the functionality inside this new project.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/Commands/UninstallHooks.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+     * @var string
+     */
+    protected $signature = 'hooks:uninstall';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Remove all hooks inside .git/hooks/';
```
Comment (mr-feek) [inline src/Commands/UninstallHooks.php]: nitpick, but could you reword this to "Remove all hooks generated by the laravel git hooks package"?

**Review by mr-feek — APPROVED** [review]
thanks!

File: src/Commands/UninstallHooks.php (hunk 1 above)
Comment (listefano) [reply src/Commands/UninstallHooks.php]: No problem, just pushed a change with a new description.
